great_expectations/rule_based_profiler/parameter_builder/parameter_container.py

```diff
@@ -200,8 +200,11 @@ def _convert_dictionaries_to_parameter_nodes(
 
         return source
 
+    def to_dict(self) -> dict:
+        return asdict(self)
+
     def to_json_dict(self) -> dict:
-        return convert_to_json_serializable(data=asdict(self))
+        return convert_to_json_serializable(data=self.to_dict())
 
```

Comment:
Should a more clear distinction be made between these two? Perhaps `to_raw_dict`? Or a docstring to clarify the difference/usage?

Reply:
@cdkini Not sure if we need another docstring here -- all it means is that `to_dict()` is an interface method as is `to_json_dict()` -- the two have sufficiently different spelling, so the chance for confusion is low.  The reason for `to_raw_dict()` elsewhere was to avoid name collision with the interface method that behaves differently.  Thanks.

Reply:
that makes sense to me thanks! LGTM